python/taichi/lang/ops.py

```diff
@@ -579,3 +579,22 @@ def rescale_index(a, b, I):
         if a.shape[n] < b.shape[n]:
             Ib.entries[n] = I.entries[n] * (b.shape[n] // a.shape[n])
     return Ib
+
+
+def get_addr(f, indices):
```

Comment:
Sorry my bad, we actually follow the Google docstring style... See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Understood. will fix.